### PR TITLE
GRAD2-3119

### PR DESCRIPTION
**Pull request by cditcher**
Updates include:
DeploymentConfig -> Deployment
Routes with certificates
Configmap update script for backend

### CODE REVIEW

File: .github/workflows/deploy-frontend-to-ocp-prod.yml [hunk 1]
```diff
@@ -37,7 +33,7 @@ env:
   MAX_REPLICAS: "5"
 
   # SITE_URL should have no scheme or port. It will be prepended with https://
-  HOST_ROUTE: ${{ secrets.SITE_URL }}
+  HOST_ROUTE: "educ-grad-admin-bbe4c3-prod.apps.silver.devops.gov.bc.ca"
```
Comment (mightycox) [inline .github/workflows/deploy-frontend-to-ocp-prod.yml:36]: Is it ok to have these in the repo?

File: backend/src/routes/auth.js [hunk 2]
```diff
@@ -54,34 +55,33 @@ function logout(req) {
...
+  }else {
     let retUrl;
     if (req.query && req.query.sessionExpired) {
-      retUrl = encodeURIComponent(
-        config.get("logoutEndpoint") +
-          "?post_logout_redirect_uri=" +
-          config.get("server:frontend") +
-          "/session-expired"
-      );
+      console.log("1")
```
Comment (mightycox) [inline backend/src/routes/auth.js:78]: console logs needed?

File: tools/openshift/update-configmap-backend.sh [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  --from-literal=TOKEN_EXPIRES_IN='1800000' \
+  --from-literal=UI_PUBLIC_KEY="$UI_PUBLIC_KEY_VAL" \
+  --from-literal=UI_PRIVATE_KEY="$UI_PRIVATE_KEY_VAL" \
+  --from-literal=GRAD_ROLE_ADMIN=GRAD_SYSTEM_COORDINATOR \
+  --from-literal=GRAD_PROGRAM_AREA_BA=GRAD_PROGRAM_AREA_BA \
+  --from-literal=GRAD_ROLE_INFO_OFFICER=GRAD_INFO_OFFICER \
+  --from-literal=REDIS_HOST=redis \
+  --from-literal=REDIS_PORT=6379 \
+  --from-literal=REDIS_PASSWORD="$REDIS_PASSWORD" \
+  --from-literal=BATCH_API_URL="http://educ-grad-batch-graduation-api.77c02f-$ENV.svc.cluster.local:8080/api/v1" \
```
Comment (mightycox) [inline tools/openshift/update-configmap-backend.sh:86]: is it ok to have namespace ids in the repo?

File: backend/src/routes/auth.js (hunk 2 above)
Comment (cditcher) [reply backend/src/routes/auth.js:78]: No probably not. This was not my change but came in when I merged main into my branch to update it. I'll notify the UI team.

File: tools/openshift/update-configmap-backend.sh (hunk 3 above)
Comment (cditcher) [reply tools/openshift/update-configmap-backend.sh:86]: I posed the question in RC to the platform team awhile back and the response was kind of luke warm about this. I have seen other teams using them in version control also which makes me think it's ok. But good catch anyway, I will change this.

**Review by mightycox — APPROVED** [review]

**Review by mightycox — APPROVED** [review]

**Pull request merged**